### PR TITLE
Update transaction-overview.md

### PR DESCRIPTION
**Pull request by cfzjywxk**
### What is changed, added or deleted? (Required)

Update the config change according to https://github.com/pingcap/docs-cn/pull/8138.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by you06 — APPROVED** [review]

**Review by jackysp — APPROVED** [review]
LGTM

File: transaction-overview.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ mysql> SELECT * FROM test;
```
Comment (ran-huang) [inline transaction-overview.md:293]: @cfzjywxk Can you help address this feedback?
> 针对 [事务限制](https://docs.pingcap.com/zh/tidb/stable/transaction-overview#%E4%BA%8B%E5%8A%A1%E9%99%90%E5%88%B6) 中描述的 「由于底层存储引擎的限制，TiDB 要求单行不超过 6 MB。可以将一行的所有列根据类型转换为字节数并加和来估算单行大小。」目前已经支持参数进行调整，需要增加参数来告诉用户该如何调整，且务必需要说明，调整改参数可能带来的后果，譬如超出 raftstore message 的大小，或者 grpc send msg 的大小

Comment (cfzjywxk) [reply transaction-overview.md:293]: It is a much more complex topic, it's better to do the enhancement task seperately.

Comment (ran-huang) [reply transaction-overview.md:293]: ok

**Review by ran-huang — APPROVED** [review]

**Pull request merged**